modules/web_hosting/iam.tf

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#########################################################################
+# IAM - Trusted User/Group
+#########################################################################
+
+resource "google_project_iam_member" "trusted_user_group_role1" {
```

Comment:
Can we update naming to sth more meaningful, instead of `role1`? :).  We can include the role name in the name of the resource

Reply:
Done! Included as part of https://github.com/GoogleCloudPlatform/rad-lab/pull/79/commits/227c406736b41c59aeeb6d22250818bac7a9522b.